pkg/itests/ingest_test.go

```diff
@@ -139,3 +141,82 @@ func TestParseEmptyCSV(t *testing.T) {
 
 	}
 }
+
+func TestIngestPolyHint(t *testing.T) {
+	f, err := os.Open("testdata/fhir/schemas/fhir.schema.json")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer f.Close()
+
+	b, err := bundle.LoadBundle("testdata/fhir/schemas/fhir.bundle.yaml", func(parentBundle string, loadBundle string) (bundle.Bundle, error) {
+		var bnd bundle.Bundle
+		if err := cmdutil.ReadJSONOrYAML(loadBundle, &bnd); err != nil {
+			return bnd, err
+		}
+		return bnd, nil
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if err := b.Build(ls.DefaultContext(), func(ctx *ls.Context, fname string) ([][][]string, error) {
+		return cmdutil.ReadSheets(fname)
+	}, func(ctx *ls.Context, fname string) (io.ReadCloser, error) {
+		return os.Open(fname)
+	}, func(ctx *ls.Context, fname string) (*ls.Layer, error) {
+		data, err := os.ReadFile(fname)
+		if err != nil {
+			return nil, err
+		}
+		var v interface{}
+		err = json.Unmarshal(data, &v)
+		if err != nil {
+			return nil, err
+		}
+		return ls.UnmarshalLayer(v, ctx.GetInterner())
+	}); err != nil {
+		t.Fatal(err)
+	}
+
+	layer, err := b.GetLayer(ls.DefaultContext(), "https://hl7.org/fhir/Patient")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	parser := jsoningest.Parser{Layer: layer}
+
+	builder := ls.NewGraphBuilder(nil, ls.GraphBuilderOptions{
+		EmbedSchemaNodes: true,
+	})
+
+	_, err = jsoningest.IngestStream(ls.DefaultContext(), b.Base, f, parser, builder)
```

Comment:
You are ingesting a schema, not data. Get one of the synthea sample files put it under testdata and ingest that